pages/docs/protocols/dkg/key-rotation.mdx

```diff
@@ -17,3 +19,11 @@ next authorities are selected.
 4. Once this signature is posted, anyone can propagate it.
    - Any relayer.
    - Any user who wants to update the governor of their contract.
+
+# Key rotation flow
+
+The onchain keys are rotate every session, this is done so that the dkg validators and network validators are aligned. At the end of the session, the
+dkg-gadget triggers the process to generate a new key. The new key is generated by the new onchain authorities, these authorities then work together
+to generate a new key and signature.
```

Comment:
```suggestion
The on-chain keys are rotate every session, this is done so that the DKG validators and network validators are aligned. At the end of the session, the `dkg-gadget` triggers the process to generate a new key. The new key is generated by the new on-chain authorities, these authorities then work together to generate a new key and signature.
```